src/util/address.rs

```diff
@@ -397,11 +401,11 @@ impl Payload {
             Payload::ScriptHash(ScriptHash::from_inner(hash_inner))
         } else if script.is_witness_program() {
             Payload::WitnessProgram {
-                version: WitnessVersion::from_opcode(opcodes::All::from(script[0])).ok()?,
+                version: WitnessVersion::from_opcode(opcodes::All::from(script[0])).ok().unwrap(),
```

Comment:
Typically we never want to use `unwrap` in non-test code because it panics. If the code path can never be hit use `expect` and explain why it cannot panic in the argument string. 

(I'm intentionally _not_ putting a suggestion for a better way to do this so you can problem solve it yourself :)

Reply:
Makes sense. Let me know if the current change looks good